### DISCUSSION TITLE
pnpm add hangs

### DISCUSSION THREAD

**Discussion by thatcort** [pnpm/pnpm#3266] [Q&A]
When I run `pnpm add ___` it hangs without completing. For example:

```
❯ pnpm add @truffle/contract@latest
Progress: resolved 1054, reused 1054, downloaded 0, added 0  
```

Then it just hangs there. Is there a way to figure out why it's stopped working?

**Comment by thatcort** [comment]
Got it working by editing the package.json file. No idea what it didn't like, but manually updating all entries to their latest versions fixed things.

**Reply by zkochan** [reply]
Do you use Windows? Might be caused by graceful-fs. In pnpm v6 we remove graceful-fs from our dependencies.

**Reply by thatcort** [reply]
Yes, that was on Windows. Good to know. Thanks.
